src/odemis/driver/xt_client.py

```diff
@@ -79,6 +111,7 @@ def move_stage(self, position, rel=False):
             position. If False the stage is moved to the absolute position.
         """
         self.client.move_stage(position, rel)
+        return self.get_stage_position()
```

Comment:
I wouldn't recommend putting the `get*()` into the `set*()`. It's handy to have direct wrappers. Inside the Scanner component, you'd create special setter methods, which do both the call to the set() and get().